content/blog/announcing-enum-support/index.md

```diff
@@ -0,0 +1,362 @@
+
+![ENUM_ERROR_TYPESCRIPT](ts-enum-spelling.png)
+
+{{< /choosable >}}
+{{< /chooser >}}
+
+### "Relaxed" enums
+
+When a property is a "relaxed" enum, the property type is specified as the `Union` of the enum type and the underlying primitive type. This means that you have the convenience of using the enum constants (i.e., `s3.CannedAcl.Private`), but you may also pass in the raw string (i.e., `"private"`).
+
+In the AWS provider (and other Terraform-based providers), we have opted for **only** using "relaxed" enums. The reasoning here is twofold. Allowing the primitive type maintains backward compatibility and also allows users to use values that may not yet be represented in the Pulumi schema (e.g., a new Managed Policy ARN, EC2 Instance type, etc.).
+
+{{< chooser >}}
+{{< choosable language typescript >}}
+
+```typescript
+export class Bucket extends pulumi.CustomResource {
+    constructor(name: string, args?: BucketArgs, opts?: pulumi.CustomResourceOptions) {...}
+}
+
+export interface BucketArgs {
+    readonly acl?: pulumi.Input<string | enums.s3.CannedAcl>;
+    ...
+}
+```
+
+{{< /choosable >}}
+{{< choosable language python >}}
+
+```python
+class Bucket(pulumi.CustomResource):
+    def __init__(__self__,
+                 resource_name: str,
+                 acl: Optional[pulumi.Input[Union[str, 'CannedAcl']]] = None,
+                 ...):
+        ...
+
+```
+
+{{< /choosable >}}
+{{< choosable language csharp >}}
+
+```csharp
+namespace Pulumi.Aws.S3
+{
+    public partial class Bucket : Pulumi.CustomResource
+    {
+        public Bucket(string name, BucketArgs? args = null, ...)
+        {
+        }
+    }
+
+    public sealed class BucketArgs : Pulumi.ResourceArgs
+    {
+        [Input("acl")]
+        public InputUnion<string, Pulumi.Aws.S3.CannedAcl>? Acl { get; set; }
+
+        ...
+    }
+}
+```
+
+{{< /choosable >}}
+{{< choosable language go >}}
+
+```go
+func NewBucket(ctx *pulumi.Context,
+name string, args *BucketArgs, ...) (*Bucket, error) {
+    ...
+}
+
+type BucketArgs struct {
+    Acl pulumi.StringPtrInput
+    ...
+}
+```
+
+{{< /choosable >}}
+{{< /chooser >}}
+
+## Try them out!
+
+You can find enum types integrated into `v3.19.0` of the [AWS provider](https://www.pulumi.com/docs/reference/pkg/aws/) and `v0.3.0` of the [Azure NextGen provider](https://www.pulumi.com/docs/reference/pkg/azure-nextgen/), and we will be adding enums to other providers in the coming weeks and months.
+
+**Azure NextGen**: In the Azure NextGen provider, all properties labeled as enums in the OpenAPI spec are represented as such. In all, there are over 1300 enums provided in the SDKs. The Azure NextGen provider uses "strict" enums since the OpenAPI specification explicitly defines its properties as such.
```

Comment:
> The Azure NextGen provider uses "strict" enums since the OpenAPI specification

Am I misreading this? Most enums in Azure NG are actually relaxed.

Reply:
Yeah I guess it's unclear because of the refactoring that @lukehoban requested. I can just take it out. I was trying to make a point that NG is the only place strict enums are used currently.

Reply:
I clarified that it uses both. Let me know if you still see any issues.